application/pr/viewer.jsx

```diff
@@ -108,7 +124,7 @@ export default class PRViewer extends React.Component {
                 }
                 action = <span>{revert}{ revert && merge ? " — or " : (merge ? " — " : "")}{merge}</span>;
             } else {
-                action = <span><button  onClick={this.revalidate.bind(this)}>Revalidate</button> <button name="nonsubstantive" onClick={this.markSubstantiveOrNot.bind(this)}>Mark as non-substantive</button></span>;
+                action = <span><button  onClick={this.revalidate.bind(this)}>Revalidate</button> <button name="nonsubstantive" onClick={this.markSubstantiveOrNot.bind(this)}>Mark as non-substantive</button> { this.props.isAdmin ? <button name="nplc" onClick={this.askNPLC.bind(this)}>Ask for non-participant commitment</button> : "" }</span>;
```

Comment:
Right now the button will only appear for people with admin rights on ash-nazg - shouldn't it also appear for anyone who is logged in and is team contact of one of the groups "owning" the repo?

Reply:
Another test before displaying that button should be that the repo is owned by Working Groups. The non-participant licensing commitment doesn't make sense for CGs.

Reply:
I've added the test for CG vs BG. The text will be slightly different depending on the type of group.
Regarding the test on the team contact, I looked a bit and this can be tricky. If the team contact hasn't linked his W3C and GH accounts, we won't be able to find his WGs from the W3C API. And if he has done so, shouldn't he be listed as admin in the repo manager?

Reply:
only very people get admin in repo manager (think of it as the kind of privileges systeam gets on W3C systems, not the kind regular staff gets)

I think it's OK that all staff contacts that haven't done what is expected don't get the button; but those that have should.

Reply:
I've updated the PR so that team contacts that aren't admin should see the button as well: https://github.com/w3c/ash-nazg/pull/163/commits/f7e400aaa58531f5ffe3e1b932b55b4c34117602

Note, it depends on https://github.com/w3c/node-w3capi/pull/42. If everything looks good, I'll tag `node-w3capi` after the PR gets merged and I'll bump the version in `package.json`

Reply:
since we're converging now, I think merging the node-w3capi PR and bumping the package.json dependency in this PR would make sense now